shared/rakefiles/xk.rake

```diff
@@ -65,6 +65,37 @@ task :xk, [:cmd, :skip_infra, :skip_secret_mgmt] => [@serviceaccount_key_file, @
   sh_filter "#{@exekube_cmd} #{args[:cmd]}" if args[:cmd]
 end
 
+task :refresh_infra, [:project_type] => [@gcp_creds_file] do | taskname, args|
```

Comment:
I find this name confusing since each gcp/live environment has its own `infra` directory. How about `refresh_common_infra`?